src/pages/DetailsPage.js

```diff
@@ -76,7 +76,7 @@ const DetailsPage = ({
                             <Text style={[styles.displayName, styles.mt1, styles.mb6]} numberOfLines={1}>
                                 {details.displayName && isSMSLogin
                                     ? toLocalPhone(details.displayName)
-                                    : details.displayName}
+                                    : details.displayName || null}
```

Comment:
This whole line is a bit crazy and precedence of operations is super hard to read... can you add some parenthesis to make this clearer please?
Like:
```suggestion
                                    : (details.displayName || null)}
```
But not sure that's what you meant.